spec/cargonextest_spec.vim

```diff
@@ -0,0 +1,159 @@
+source spec/support/helpers.vim
+
+describe "CargoNextest"
+
+  before
+    cd spec/fixtures/cargo/crate
+  end
```

Comment:
Could you please add the following back in and I'll see if I can figure a way to avoid this in the future on CI

```viml
let g:test#rust#runner = 'cargonextest'
```